setup.sh

```diff
@@ -1,22 +1,26 @@
-#!/bin/bash
-Windows=0;
+#! /usr/bin/env bash
 
-while [[ "$#" -gt 0 ]]; do case $1 in
-  --windows) 
-    Windows=1;;
-esac; shift; done
+# Check which Python versions are available
+python --version                                                                                                                                                                                             python --version
```

Comment:
Something strange must have happened with the copy-paste, we should remove all of the commands that have duplicated